tests/acl/test_acl.py

```diff
@@ -1307,7 +1307,7 @@ def post_setup_hook(self, dut, localhost, populate_vlan_arp_entries, tbinfo, con
         # We need some additional delay on e1031
         if dut.facts["platform"] == "x86_64-cel_e1031-r0":
             time.sleep(240)
-        if 't1' in tbinfo["topo"]["name"]:
+        if 't1' in tbinfo["topo"]["name"] or 'm0' in tbinfo["topo"]["name"]:
```

Comment:
@bingwang-ms Can we remove this `if` condition and apply BGP check to all topology?

Reply:
My concern is there are no BGP neighbors on some topo, and the issue was only seen in T1. That's why I added the check of topo.